### ISSUE TITLE
Adding jackets to the card draw

### ISSUE THREAD

**Issue by FuriousDCSL** [noahm/DDRCardDraw#5]
I've been working on adding jackets to my fork. I have got it to work, only added jackets for the 19s for testing.

![image](https://user-images.githubusercontent.com/39023936/47770977-b9814380-dc9e-11e8-82e0-2ca6d09069f2.png)

It would be a lot of data entry to add all the jackets. I am willing to work on it if it will get used. Let me know

**Comment by noahm** [comment]
Oh hell yeah, you better believe it will get used! Adding jackets has been on my own personal wish-list for a long time. That said, I don't think it's worth your time to do every single one manually. I have an idea for grabbing all of them automatically with a script, but I haven't spent the time getting it working yet.

I'm curious to see what changes you've made so far to get what you have in the screenshot, though. Can you push your changes up to your fork for me to take a look at?

**Comment by FuriousDCSL** [comment]
Have to drop contents of Jackets into the /var/www/html/dist folder

**Comment by FuriousDCSL** [comment]
Ok wrote a script to import most jackets. need to manually do recent jackets and a few songs with commas in the title. 
![image](https://user-images.githubusercontent.com/39023936/47816746-29c8ad00-dd11-11e8-805d-623592629507.png)

Found a couple errors in song name (fixed)

 found a couple missing japan exclusives. will fix these soon

Added a folder tag while i was at it.

**Comment by FuriousDCSL** [comment]
I wrote the script in python i have them in my fork now. Fork is updated. The script parses a csv dump of a database that is several month old so i had to manually add new songs since then. also had to manually ad any song with comma in the name as it messed up the parsing of the csv.

**Comment by FuriousDCSL** [comment]
Made the my fork live at https://furiousdcsl.github.io/

**Comment by noahm** [comment]
Just merged it in and deployed. I added you to the credits for all the help, but let me know if you'd rather be credited a different way.

**Comment by FuriousDCSL** [comment]
Cool!  Credits are fine.  When you changed the cards to background with the cover size it buts the top of folders off on widescreen displays. Changed size to auto for my version.  

![image](https://user-images.githubusercontent.com/39023936/47933960-455bc100-de93-11e8-9630-2d6f2888f0d1.png)

Top is cover, bottom is auto, neither looks great i guess. 

I'm starting to work on the extreme banners.

**Comment by noahm** [comment]
Yeah, I tried a lot of different options for how to best integrate the images and ultimately I decided that `background-size: cover` had the nicest overall look while still remaining recognizable. I don't feel a strong need to have the entire image be visible because to my mind the purpose of the jackets is just to help jog the memory a bit when the title alone doesn't always do the trick. I'll be keeping an eye on community reaction to the jackets and continuing to tweak with any better ideas that come up.
